### PR TITLE
SC-2040: create schema for postgres db

### PR DESCRIPTION
**Pull request by akashkj**
Changes to create schema in the DB. This is a sub-task under automatic roles creation US.

TODO - Adding unit test for postgres specific code, I'm looking at some mock utility lib that can be used for postgres testing.

### CODE REVIEW

File: hq_superset/model/db_info.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class DbInfo(object):
```
Comment (sravfeyn) [inline hq_superset/model/db_info.py:5]: Can we not use the URL style connection string to avoid this class?

**Review by proteusvacuum — CHANGES_REQUESTED** [review]

**Review by kaapstorm — COMMENTED** [review]
It looks like we are trying to reinvent a very well-established wheel. Is there a reason we aren't using SQLAlchemy + Alembic [as Superset already does](https://github.com/apache/superset/blob/2631558ac4ba596b24876a9b0702bbf3a6bfbb38/superset/migrations/alembic.ini#L20) for managing our database?

**Comment by sravfeyn** [comment]
Actually, this could be much simpler. I have created a util here to create a schema https://github.com/dimagi/hq_superset/pull/6/commits/c35339e28dbe2130da34fda24e248b8e9094bb16

**Comment by akashkj** [comment]
That looks much simpler indeed @sravfeyn 
Will close this one

**Pull request closed without merging**